test/test_utils/test_path_utils.py

```diff
@@ -0,0 +1,58 @@
+import pathlib
+import pytest
+
+from corppa.utils.path_utils import encode_htid, decode_htid
+from corppa.utils.path_utils import get_stub_dir, get_vol_dir
+
+
+def test_encode_htid():
```

Comment:
these unit tests look great - nice to have confirmed functionality and expected error cases